content/applications/inventory_and_mrp/inventory/barcode/operations/gs1_usage.rst

```diff
@@ -0,0 +1,234 @@
+=================
+GS1 barcode usage
+=================
+
+.. _barcode/operations/gs1_usage:
+
+.. |AI| replace:: :abbr:`A.I. (Application Identifier)`
+.. |GTIN| replace:: :abbr:`GTIN (Global Trade Item Number)`
+
+GS1 barcodes are a standardized language for barcode scanners helping businesses with international
+supply chain operations. Use Odoo *Barcode* to interpret and print GS1 barcodes, making warehouse
+operations, like receiving and picking, more efficient.
+
+The following sections contain examples of how Odoo uses GS1 barcodes provided by the business to
+identify common warehouse items and automate certain warehouse workflows.
+
+.. important::
+   Odoo **does not** create GS1 barcodes. Businesses must purchase a unique Global Trade Item Number
+   (GTIN) from GS1. Then, they can combine their existing GS1 barcodes with product and supply chain
+   information (also provided by GS1) to create barcodes in Odoo.
+
+.. seealso::
+   - `Purchase GTINs <https://www.gs1.org/standards/get-barcodes>`_
+   - :ref:`GS1 nomenclature <barcode/operations/gs1>`
+
+.. _barcode/operations/gs1-lots:
+
+Configure barcodes for product, quantity, and lots
+==================================================
+
+To build a GS1 barcode for a box that contains a product, number of units in it, and the lot number,
```

Comment:
The way this is phrased implies that all three variables are necessary in a GS1 barcode. I don't think they are?

What about information like batch number, expiration dates, etc., which can also be included in GS1's?

I'd rephrase this intro sentence so it's not forcing a specific scenario:

```suggestion
To build a GS1 barcode for a shipment, use the following barcode patterns and Application Identifiers (A.I.) in the table below:
```

For a future PR, especially since the apple is a good example: I wonder if you can include expiration dates too here?